<!DOCTYPE 
    HTML PUBLIC 
    "-//W3C//DTD HTML 4.01 Transitional//EN" 
    "http://www.w3.org/TR/html4/loose.dtd"
    >

<!--

//
// Copyright (C) Julian I. Kamil <julian.kamil@gmail.com>
// No warranty is provided.  Use at your own risk.
//
// Commercial support is available through ESV Media Group
// who can be reached at: http://www.ESV-i.com/.
//
// Name: simple.tmpl
// Author: Julian I. Kamil <julian.kamil@gmail.com>
// Created: 2005-05-18
// Description:
//     This is a simple skin for PmWiki. Please see:
//         http://www.madhckr.com/project/PmWiki/SimpleSkin
//     for a live example and doumentation.
//
// $Id: simple.tmpl,v 1.1 2005/08/17 19:24:54 julian Exp $
//
// History:
//     2005-05-18  jik  Created.
//     2005-06-21  jik  Added the directive that enables
//                      the injection of additional stylesheets.
//     2005/06/27  jik  Added SkinTone feature.
//     2005/07/06  jik  Added group name in the title.
//     2005/07/15  jik  Added code to style abbr on IE.
//

-->

<html>
    <head>
        <title>杜朴风博士的个人主页 : Main - Home Page </title>
        <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
        <!--HeaderText--><style type='text/css'><!--
  ul, ol, pre, dl, p { margin-top:0px; margin-bottom:0px; }
  code.escaped { white-space: nowrap; }
  .vspace { margin-top:1.33em; }
  .indent { margin-left:40px; }
  .outdent { margin-left:40px; text-indent:-40px; }
  a.createlinktext { text-decoration:none; border-bottom:1px dotted gray; }
  a.createlink { text-decoration:none; position:relative; top:-0.5em;
    font-weight:bold; font-size:smaller; border-bottom:none; }
  img { border:0px; }
  
  .rtl {direction:rtl; unicode-bidi:bidi-override;}
  .ltr {direction:ltr; unicode-bidi:bidi-override;}
  .rtl .indent, .rtl.indent, .rtl .outdent, .rtl.outdent {
    margin-left:0; margin-right: 40px;
  }
  .editconflict { color:green; 
  font-style:italic; margin-top:1.33em; margin-bottom:1.33em; }

  table.markup { border:2px dotted #ccf; width:90%; }
  td.markup1, td.markup2 { padding-left:10px; padding-right:10px; }
  table.vert td.markup1 { border-bottom:1px solid #ccf; }
  table.horiz td.markup1 { width:23em; border-right:1px solid #ccf; }
  table.markup caption { text-align:left; }
  div.faq p, div.faq pre { margin-left:2em; }
  div.faq p.question { margin:1em 0 0.75em 0; font-weight:bold; }
  div.faqtoc div.faq * { display:none; }
  div.faqtoc div.faq p.question 
    { display:block; font-weight:normal; margin:0.5em 0 0.5em 20px; line-height:normal; }
  div.faqtoc div.faq p.question * { display:inline; }
  td.markup1 pre { white-space: pre-wrap; }
   
    .frame 
      { border:1px solid #cccccc; padding:4px; background-color:#f9f9f9; }
    .lfloat { float:left; margin-right:0.5em; }
    .rfloat { float:right; margin-left:0.5em; }
a.varlink { text-decoration:none;}

--></style><meta http-equiv='Content-Type' content='text/html; charset=utf-8' />  <meta name='robots' content='index,follow' />

        <link rel='stylesheet' title="Simple" href='https://www.pufengdu.org/homepage/pub/skins/simple/simple.css' type='text/css' />
        <!--[if IE]>
            <link href='https://www.pufengdu.org/homepage/pub/skins/simple/simple-ie.css' rel='stylesheet' type='text/css' />
        <![endif]-->
        <script type="text/javascript">
        <!--//--><![CDATA[//><!--
        startList = 
            function() 
            {
                if (document.all && document.getElementById) {
                    navRoot = document.getElementById("menu");
                    for (i = 0; i < navRoot.childNodes.length; i++) {
                        node = navRoot.childNodes[i];
                        if (node.nodeName == "LI") {
                            node.onmouseover = 
                                function() { this.className += " over"; }
                            node.onmouseout = 
                                function() { this.className = this.className.replace(" over", ""); }
                        }
                    }
                }
            }

        function styleAbbr() 
        {
            var oldBodyText, newBodyText, reg;

            if (isIE) {
                oldBodyText = document.body.innerHTML;
                reg = /<ABBR([^>]*)>([^<]*)<\/ABBR>/g;
                newBodyText = oldBodyText.replace(reg, '<ABBR $1><SPAN class=\"abbr\" $1>$2</SPAN></ABBR>');
                document.body.innerHTML = newBodyText;
            }
        }

        window.onload = function(){
            styleAbbr();
            startList();
        };

        isIE = (document.all) ? true:false;
        //--><!]]>
        </script>
    </head>
    <body bgcolor="#ffffff">
        <div 
            id="header" 
            style="border: 1px none #ccf; background: url(https://www.pufengdu.org/homepage/pub/skins/simple/image/Box-Top-Classic.jpg) no-repeat; " 
            title='Main &raquo; Home Page was last modified on November 07, 2022, at 05:46 AM'
            >
            <table width="100%" style="padding: 0px; margin:0px;">
                <tr>
                    <td style="padding: 0px; margin:0px; border: none;">
                        <div id="page-title">杜朴风博士的个人主页</div>
                        <div id="page-subtitle">Main &raquo; Home Page</div>
                    </td>
                    <td 
                        align="right" width="120" 
                        style="padding: 0px; margin:0px; padding-top: 16px; border: none;"
                        >
                        <center>
                            <a href="https://www.pufengdu.org/homepage/pmwiki.php?n=Main"><img src="https://www.pufengdu.org/homepage/pub/skins/simple/image/icon1.png" border="0" /></a>
                        </center>
                    </td>
                </tr>
            </table>
        </div>

        <div id="main">
            <div id="menubar" style="display: none;">
                <div>
                    <span style="float: left;">
                        <ul>
                            <li><a href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.HomePage?action=edit'>Edit Page</a></li>
                            <li><a href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.HomePage?action=diff'>History</a></li>
                            
                            <li><a href='https://www.pufengdu.org/homepage/pmwiki.php?n=Site.Search'>Search</a></li>
                        </ul>
                    </span>
                    <span id="popupmenu" style="float: right;">
                        <ul id="menu">
                            <li>Website Menu
                                <ul><li><a class='selflink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.HomePage'>HomePage</a>
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.WikiSandbox'>WikiSandbox</a>
</li></ul><p class='vspace sidehead'> <a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.PmWiki'>PmWiki</a>
</p><ul><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.InitialSetupTasks'>Initial Setup Tasks</a>  
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.BasicEditing'>Basic Editing</a>
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.DocumentationIndex'>Documentation Index</a>
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.FAQ'>PmWiki FAQ</a>
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.PmWikiPhilosophy'>PmWikiPhilosophy</a>
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.ReleaseNotes'>Release Notes</a>
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.ChangeLog'>ChangeLog</a>
</li></ul><p class='vspace sidehead'> <a class='urllink' href='http://www.pmwiki.org' rel='nofollow'>pmwiki.org</a>
</p><ul><li><a class='urllink' href='http://www.pmwiki.org/wiki/Cookbook/Cookbook' rel='nofollow'>Cookbook (addons)</a>
</li><li><a class='urllink' href='http://www.pmwiki.org/wiki/Cookbook/Skins' rel='nofollow'>Skins (themes)</a>
</li><li><a class='urllink' href='http://www.pmwiki.org/wiki/PITS/PITS' rel='nofollow'>PITS (issue tracking)</a>
</li><li><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=PmWiki.MailingLists'>Mailing Lists</a>
</li></ul><p class='vspace'  style='text-align: right;'> <span style='font-size:83%'><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Site.SideBar?action=edit'>edit SideBar</a></span>
</p>

                            </li>
                        </ul>
                    </span>
                </div>
                <div class="clearer"></div>
            </div>

            <div class="content-mat" style="border: 1px none #fcc;">
                <div id="content" style="border: 1px none #ccc;">
                    <!--PageText-->
<div id='wikitext'>
<div class='img imgcaption'><span class='rfloat' style='margin: 15px; text-align: center;'> <img height='80%' width='80%' src='https://www.pufengdu.org/homepage/images/head.jpg' alt='' title='' /><br /><span class='caption'><strong>杜朴风</strong></span></span></div>
<div class='vspace'></div><h2>个人简介</h2>
<div class='indent'><strong>杜朴风</strong>
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;天津大学，智能与计算学部，计算机科学与技术学院
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;副教授，硕士研究生导师；
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;中国计算机学会(CCF)高级会员，生物信息学专业委员会，常务委员；
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;中国人工智能学会(CAAI)会员，生物信息学与人工生命专业委员会，常务委员；
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;中国自动化学会(CAA)会员，智能健康与生物信息学专业委员会，委员；
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;中国生物工程学会会员，计算生物学与生物信息学专业委员会，委员；
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;天津市药理学会，网络药理学专业委员会，副主任；
</div><div class='indent'>&nbsp;&nbsp;&nbsp;&nbsp;ACM会员，ACM SIGBIO中国执行委员会委员。
<div class='vspace'></div></div><div class='indent'><strong>现招收2023年秋季入学硕士研究生(含专业学位)，欢迎2024年秋季入学研究生提前联系</strong> <a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.%e5%92%a8%e8%af%a2'>我要咨询</a>
<div class='vspace'></div></div><div class='indent'><strong>长期招收优秀本科生加盟课题组参与科研项目。</strong>
</div><div class='vspace'></div><h2>联系方式</h2>
<div class='indent'>Email：<a class='urllink' href='mailto:pdu@tju.edu.cn' rel='nofollow'>pdu@tju.edu.cn</a> <a class='urllink' href='mailto:PufengDu@gmail.com' rel='nofollow'>PufengDu@gmail.com</a>
</div><div class='indent'>地址：天津市津南区海河教育园内雅观路135号天津大学55楼B323。
</div><div class='indent'>邮政编码：300354
</div><div class='indent'>与我联系时，请务必<strong><em>直接与我取得联系</em></strong>，发送到上述地址的<a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.%e5%ae%9e%e5%90%8d%e5%88%b6'>实名制</a>邮件，如无意外，<strong><em>会在24小时内回复</em></strong>
</div><div class='vspace'></div><h2>推荐信相关</h2>
<div class='indent'><strong>本科生出国留学<a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.%e6%8e%a8%e8%8d%90%e4%bf%a1'>推荐信</a>。</strong>
</div><div class='indent'><strong>本科生申请国内学校研究生/直博生<a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.%e5%9b%bd%e5%86%85%e6%8e%a8%e8%8d%90%e4%bf%a1'>推荐信</a>。</strong>
</div><div class='indent'><strong>外国留学生申请中国政府奖学金项目</strong><a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.InternationalStudentsCGS'>International Students CGS</a>
</div><div class='indent'><strong>研究生申请工作/读博/出国<a class='wikilink' href='https://www.pufengdu.org/homepage/pmwiki.php?n=Main.%e7%a0%94%e7%a9%b6%e7%94%9f%e6%8e%a8%e8%8d%90%e4%bf%a1'>推荐信</a>。</strong>
</div><div class='vspace'></div><h2>主要经历</h2>
<div class='indent'>2020年11月-今：	天津大学，智能与计算学部，学科团队<a class='urllink' href='http://cic.tju.edu.cn/kxyj/kytd.htm' rel='nofollow'>负责人</a>(12. 生物信息学)
</div><div class='indent'>2018年09月-今：	天津大学，智能与计算学部，副教授(独立开展科研工作)
</div><div class='indent'>2013年07月-2018年08月：	天津大学，计算机科学与技术学院，副教授(独立开展科研工作)
<div class='vspace'></div></div><div class='indent'>2012年12月-2014年12月：	香港城市大学，计算机系，访问博士后（香江学者）。合作导师：<a class='urllink' href='https://www.cs.cityu.edu.hk/~lwang/' rel='nofollow'>王鲁生</a>
<div class='vspace'></div></div><div class='indent'>2010年01月-2013年06月：	天津大学，计算机科学与技术学院，讲师(独立开展科研工作)
</div><div class='indent'>2008年09月-2010年01月：	清华大学，自动化系，李衍达院士秘书
</div><div class='indent'>2005年09月-2010年01月：	清华大学，自动化系/清华信息科学与技术国家实验室，博士研究生，2010年1月获博士学位。导师：<a class='urllink' href='http://www.au.tsinghua.edu.cn/info/1018/1709.htm' rel='nofollow'>李衍达</a>（无副导师）
</div><div class='indent'>2001年09月-2005年07月：	清华大学，自动化系，本科生，2005年7月获学士学位
</div><div class='vspace'></div><h2>研究方向</h2>
<div class='indent'><strong>基于机器学习的生物序列分析</strong>
<div class='indent'>基于图神经网络的生物组学数据分析
</div><div class='indent'>荧光图像自动读图和知识提取
</div><div class='indent'>基因组回路元件的计算识别
</div><div class='indent'>基于深度机器学习的单细胞测序数据的计算分析
</div><div class='vspace'></div></div><div class='indent'><strong>高效生物序列分析软件开发</strong>
<div class='indent'>通用生物序列特征快速抽取软件
</div><div class='indent'>超大规模序列特征可视化分析系统开发
</div><div class='vspace'></div></div><div class='indent'><strong>智能药物发现</strong>
<div class='indent'>计算机辅助的药物发现：药物重用与组合发现的计算方法
</div><div class='indent'>传统医学智能化：基于现代人工智能的中医药自动/辅助诊断与处方系统
</div></div><div class='vspace'></div><h2>部分科研项目</h2>
<div class='indent'><strong>国家自然科学基金</strong>，蛋白质异常定位组的计算预测(面上，2019-2022)，主持；
</div><div class='indent'><strong>国家自然科学基金</strong>，基于信息融合的高分辨率蛋白质定位预测(青年，2011-2013)，主持；
</div><div class='indent'><strong>国家重点研发计划</strong>，精准医学大数据的有效挖掘与关键信息技术开发(课题5，2018-2021)，骨干；
</div><div class='indent'><strong>教育部新教师基金</strong>，利用蛋白质相关性网络融合多种信息的蛋白质动态定位预测(2011-2013)，主持；
</div><div class='indent'><strong>天津市自然科学基金</strong>，融合多种异质信息的高分辨率动态蛋白质定位预测(青年，2012-2015)，主持；
</div><div class='indent'><strong>北洋学者-青年骨干教师计划</strong>，蛋白质结构与功能计算预测(2014-2015)，主持；
</div><div class='indent'><strong>天津大学自主创新基金</strong>，在基因本体子空间中针对特定拓扑蛋白的定位预测(2015-2016)，主持。
</div><div class='vspace'></div><h2>所获荣誉</h2>
<div class='indent'>2016年天津大学青年教师讲课大赛二等奖；
</div><div class='indent'>2014年天津大学“北洋学者-青年骨干教师”；
</div><div class='indent'>2013年“香江学者”奖；
</div><div class='indent'>2012年全国优秀博士学位论文（提名奖）；
</div><div class='indent'>2011年北京市优秀博士学位论文；
</div><div class='indent'>2010年天津大学“十佳优秀青年教工”（提名奖）；
</div><div class='indent'>2010年清华大学“优秀研究生”；
</div><div class='indent'>2007年全国百篇最具影响力的国际学术论文奖。
</div><div class='vspace'></div><h2>硕士研究生</h2>
<div class='indent'>2014级：焦亚森：独立指导，考研入学，获国家奖学金，优秀毕业生，校三好学生。毕业后去向：航天二院（物联网方向）。现工作单位：字节跳动。
</div><div class='indent'>2016级：张鸽：合作指导，保研入学。毕业后去向：澳大利亚Macquarie大学攻读博士学位。
</div><div class='indent'>2016级：杨晓飞：合作指导，保研入学。毕业后去向：石家庄市数据资源管理局(公务员)。
</div><div class='indent'>2017级：赵伟：独立指导，保研入学。毕业后去向：光大银行（总行信息科技部）。
</div><div class='indent'>2018级：单虹毓：合作指导，考研入学。毕业后去向：蚂蚁金服（搜索推荐平台）。
</div><div class='indent'>2018级：李光平：独立指导，考研入学，获校三好学生，毕业后去向：Bigo (live 后台开发)。
</div><div class='indent'>2018级：孙鉴：独立指导，考研入学，获校先进个人，优秀学生干部，优秀毕业生，毕业后去向：中国华录集团北京易华录信息技术有限公司(方案工程部)。
</div><div class='indent'>2018级：王俊：独立指导，考研入学（非全日制），毕业后去向：百度(移动生态评估中心)。
</div><div class='indent'>2018级：周园科：独立指导，考研入学（非全日制），毕业后去向：美团(北京，智慧交通平台)。
</div><div class='indent'>2019级：于涵：独立指导，考研入学，毕业后去向：济南铁路局信息所
</div><div class='indent'>2019级：暨巧莹：合作指导，考研入学，毕业后去向：美团（成都，到店事业群）
</div><div class='indent'>2019级：沈子昂：合作指导，考研入学，毕业后去向：腾讯（内容与平台中心）
</div><div class='indent'>2019级：张文雅：合作指导，考研入学，毕业后去向：美团（北京，基础研发平台）
</div><div class='indent'>2019级：辛晓红：独立指导，考研入学（非全日制），毕业后去向：腾讯（北京，云与智慧产业事业群）
</div><div class='indent'>2019级：高楚翘：独立指导，考研入学（非全日制），毕业后去向：自主创业。
</div><div class='indent'>2019级：闵慧：独立指导，考研入学（非全日制），毕业后去向：武汉联影科技医疗有限公司（测试工程师）
</div><div class='indent'>2019级：刘航宇：独立指导，考研入学，在读（延期中...）。
</div><div class='indent'>2020级：张颖颖：独立指导，保研入学，在读。
</div><div class='indent'>2020级：刘思凯：合作指导，保研入学，在读。
</div><div class='indent'>2020级：王仁华：合作指导，保研入学，在读。
</div><div class='indent'>2020级：吴杨：独立指导，考研入学，在读。
</div><div class='indent'>2021级：温靖雯：独立指导，保研入学，在读。
</div><div class='indent'>2021级：潘建华：独立指导，保研入学，在读。
</div><div class='indent'>2021级：王朴石：合作指导，保研入学，在读。
</div><div class='indent'>2021级：孙睿：独立指导，考研入学，在读。
</div><div class='vspace'></div><h2>课题组本科生</h2>
<div class='indent'>缪阳洋，2015级，化工学院。毕业后去向：中国科学技术大学（合肥）读研。
</div><div class='indent'>李皓民，2018级，智能与计算学部（腾讯班）。毕业后去向：上海交通大学，直博。
</div><div class='vspace'></div><h2>本科生教学</h2>
<div class='indent'><strong>校必修基础课</strong>
<div class='indent'>C++程序设计与算法基础(英)(2017-2020)；
</div><div class='indent'>大学计算机基础(2011,2012,2015-今)；
</div><div class='indent'>计算机软件技术基础(C++版)(2015, 2016, 2020-今)；
</div><div class='indent'>程序设计原理(双)(求是学部)(2021-今)；
</div></div><div class='indent'><strong>校选修基础课</strong>
<div class='indent'>Python程设设计及应用(英)(2018)；
</div><div class='indent'>Visual C++程序设计(2015-2018)；
</div></div><div class='indent'><strong>院选修专业课</strong>
<div class='indent'>开源技术及应用(2011, 2012, 2015-2018)；
</div></div><div class='vspace'></div><h2>学术服务</h2>
<div class='indent'><strong>期刊编委</strong>
<div class='indent'>高等教育出版社：<em>Frontiers of Computer Science</em> (青年编委)
</div><div class='indent'>MDPI出版社：<em>Data</em>（编委）
</div><div class='indent'>Bentham Science：<em>Current Bioinformatics</em>（Editorial Board Member / Section Editor）
</div><div class='indent'>Bentham Science: <em>Current Chinese Computer Science</em> (Editorial Board Member)
</div><div class='indent'>Bentham Science: <em>Current Gene Therapy</em> (Editorial Board Member)
</div><div class='indent'>Frontiers: <em>Fronteirs in Genetics</em> (Editorial Board Member / Review Editor)
</div><div class='vspace'></div></div><div class='indent'><strong>专刊编委</strong>
<div class='indent'><em>Current Organic Chemistry</em>, Thematic Issue：Computational Modelling of Biological Macromolecule (2019年4月30日截稿)
</div><div class='indent'>MDPI-<em>Data</em>, Special Issue： <a class='urllink' href='https://www.mdpi.com/journal/data/special_issues/Bioinform_datasets' rel='nofollow'>Benchmarking datasets in Bioinformatics</a> (2020年7月31日截稿)
</div></div><div class='indent'><strong>论文评阅人</strong>
<div class='indent'>Nature Communications; Briefings in Bioinformatics; Bioinformatics; BMC Bioinformatics; BMC Genomics; BMC Supplements; BMC Medical Informatics and Decision Making; IEEE/ACM Transactions on Computational Biology and Bioinformatics; Journal of the Royal Statistical Society - Serial A; IEEE Access; Journal of Proteome Research; Expert Review of Proteomics; RNA Biology; Molecular Therapy - Nucleic Acids: Journal of Biotechnology; Journal of Cellular and Molecular Medicine; Current Bioinformatics; Current Genomics; Current Proteomics; Letters in Organic Chemistry; Evolutionary Bioinformatics; Genomics; Amino Acids; Plos ONE; Scientific Reports; Oncotarget; NeuroComputing; Molecular Informatics; Molecular BioSystems; Applied Sciences; Interdisciplinary Sciences: Computational Life Sciences; BioTechnologia; International Journal of Biomathematics; Acta BioTheoratica; Artificial Intelligence in Medicine; Computational and Mathematical Methods in Medicine; Computer Methods and Programs in Biomedicine; BioMed Research International; IET Biometrics; Chemometrics and Intelligent Laboratory Systems; PeerJ; Journal of Theoretical Biology; Protein and Peptides Letters; Frontiers in Genetics; Frontiers of Computer Sciences; Genomics, Proteomics &amp; Bioinformatics; Quantitative Biology; Progress in Biochemistry and Biophysics; MDPI - Molecules; MDPI - Data; MDPI - Information; MDPI - Genes;
</div></div><div class='indent'><strong>会议PC</strong>
<div class='indent'>WABI2012; APCIIT2013; ICSEA2016; GIW2016; BIIP2016; CBC2016;  BIBM2016(Workshop); ICIC2017; BIIP2017; CBC2017; NCCA2017; ICIC2018; ISBRA2018; BIIP2018 (Org-Chair); CBC2018; NCCA2018; SMC2018; ISB2018; ISBRA2019; ICPCSEE2019; CBC2019; BIIP2019; ICBBE 2019; ICIC2020; ISBRA2020;ICIC2022;ISBRA2022;BIBM2022;CBC2022
</div></div><div class='vspace'></div><h2>代表论文</h2>
<div class='indent'><strong>完整列表请参阅 <a target='_blank'  class='urllink' href='https://scholar.google.com/citations?user=EHvA-IUAAAAJ&amp;hl=en' rel='nofollow'>Google Scholar</a>页面</strong>
</div><div class='vspace'></div><div class='outdent'>[31]&nbsp;&nbsp;Ying-Ying Zhang, Wen-Ya Zhang, <strong>Pu-Feng Du<sup>*</sup></strong>; dbEssLnc: A manually curated database of human and mouse essential lncRNA genes, <em>Computational and Structural Biotechnology Journal</em>, 2022, <strong>(SCI)</strong>
</div><div class='outdent'>[30]&nbsp;&nbsp; Zi-Ang Shen, Tao Luo, Yuan-Ke Zhou, Han Yu, <strong>Pu-Feng Du<sup>*</sup></strong>; NPI-GNN: Predicting ncRNA–protein interactions with deep graph neural networks, <em>Briefings in Bioinformatics</em>, 2021, DOI:10.1093/bib/bbab051. <strong>(SCI)</strong>
</div><div class='outdent'>[29]&nbsp;&nbsp; Wen-Ya Zhang; Junhai Xu<sup>*</sup>; Jun Wang; Yuan-Ke Zhou; Wei Chen<sup>*</sup>; <strong>Pu-Feng Du<sup>*</sup></strong>; KNIndex: a comprehensive database of physicochemical properties for k-tuple nucleotides, <em>Briefings in Bioinformatics</em>, 2021, DOI:10.1093/bib/bbaa284. <strong>(SCI)</strong>
</div><div class='outdent'>[28]&nbsp;&nbsp; Guang-Ping Li; <strong>Pu-Feng Du<sup>*</sup></strong>; Zi-Ang Shen; Hang-Yu Liu; Tao Luo<sup>*</sup>; DPPN-SVM: Computational Identification of Mis-Localized Proteins in Cancers by Integrating Differential Gene Expressions With Dynamic Protein-Protein Interaction Networks, <em>Frontiers in Genetics</em>, 2020, 11: 600454. <strong>(SCI)</strong>
</div><div class='outdent'>[27]&nbsp;&nbsp; Jian Sun; <strong>Pu-Feng Du<sup>*</sup></strong>; Predicting protein subchloroplast locations: the 10th anniversary, <em>Frontiers of Computer Science</em>,2021, 15(2): 152901. <strong>(SCI)</strong>
</div><div class='outdent'>[26]&nbsp;&nbsp;Jun Wang; <strong>Pu-Feng Du<sup>*</sup></strong>; Xin-Yu Xue; Guang-Ping Li; Yuan-Ke Zhou; Wei Zhao; Hao Lin; Wei Chen<sup>*</sup>; VisFeature: a stand-alone program for visualizing and analyzing statistical features of biological sequences, <em>Bioinformatics</em>, 2020, 36(4): 1277-1278. <strong>(SCI)</strong>
</div><div class='outdent'>[25]&nbsp;&nbsp;Yuan-Ke Zhou; Zi-Ang Shen; Han Yu; Tao Luo<sup>*</sup>; Yang Gao<sup>*</sup>; <strong>Pu-Feng Du<sup>*</sup></strong>; Predicting lncRNA–Protein Interactions With miRNAs as Mediators in a Heterogeneous Network Model, <em>Frontiers in Genetics</em>, 2020, 10: 1341. <strong>(SCI)</strong>
</div><div class='outdent'>[24]&nbsp;&nbsp;Yang-Yang Miao; Wei Zhao; Guang-Ping Li; Yang Gao<sup>*</sup>; <strong>Pu-Feng Du<sup>*</sup></strong>; Predicting endoplasmic reticulum resident proteins using auto-cross covariance transformation with U-shaped residue weight transfer function, <em>Frontiers in Genetics</em>, 2019, 10: 1231. <strong>(SCI)</strong>
</div><div class='outdent'>[23]&nbsp;&nbsp;Wei Zhao; Guang-Ping Li; Jun Wang; Yuan-Ke Zhou; Yang Gao<sup>*</sup>; <strong>Pu-Feng Du<sup>*</sup></strong>; Predicting protein sub-Golgi locations by combining functional domain enrichment scores with pseudo-amino acid compositions, <em>Journal of Theoretical Biology</em>, 2019, 473: 38-43. <strong>(SCI)</strong>
</div><div class='outdent'>[22]&nbsp;&nbsp;Wei Zhao; Likun Wang; Tian-Xiang Zhang; Ze-Ning Zhao; <strong>Pu-Feng Du<sup>*</sup></strong>; A brief review on software tools in generating Chou's pseudo-factor representations for all types of biological sequences, <em>Protein and Peptide Letters</em>, 2018, 25(9): 822-829. <strong>(SCI)</strong>
</div><div class='outdent'>[21]&nbsp;&nbsp;<strong>Pu-Feng Du</strong>; Wei Zhao; Yang-Yang Miao; Le-Yi Wei; Likun Wang<sup>*</sup>; UltraPse: a universal and extensible software platform for representing biological sequences, <em>International Journal of Molecular Sciences</em>, 2017, 18(11): 2400. <strong>(SCI)</strong>
</div><div class='outdent'>[20]&nbsp;&nbsp;<strong>Pu-Feng Du<sup>*</sup></strong>; Predicting protein submitochondrial locations: The 10th Anniversary, <em>Current Genomics</em>, 2017, 18(4): 316-321. <strong>(SCI)</strong>
</div><div class='outdent'>[19]&nbsp;&nbsp;Ya-Sen Jiao; <strong>Pu-Feng Du<sup>*</sup></strong>; Predicting protein submitochondrial locations by incorporating the positional-specific physicochemical properties into Chou's general pseudo-amino acid compositions, <em>Journal of Theoretical Biology</em>, 2017, 416: 81-87. <strong>(SCI)</strong>
</div><div class='outdent'>[18]&nbsp;&nbsp;Yasen Jiao; <strong>Pufeng Du<sup>*</sup></strong>; Performance measures in evaluating machine learning based bioinformatics predictors for classifications, <em>Quantitative Biology</em>, 2016, 4(4): 320-330.
</div><div class='outdent'>[17]&nbsp;&nbsp;Ya-Sen Jiao; <strong>Pu-Feng Du<sup>*</sup></strong>; Prediction of Golgi-resident protein types using general form of Chou's pseudo-amino acid compositions: Approaches with minimal redundancy maximal relevance feature selection, <em>Journal of Theoretical Biology</em>, 2016, 402: 38-44. <strong>(SCI)</strong>
</div><div class='outdent'>[16]&nbsp;&nbsp;<strong>Pu-Feng Du<sup>*</sup></strong>; A brief review on software implementations and algorithm enhancements of Chou’s pseudo-amino acid compositions, <em>Current Proteomics</em>, 2016, 13(2): 105-112. <strong>(SCI)</strong>
</div><div class='outdent'>[15]&nbsp;&nbsp;Ya-Sen Jiao; <strong>Pu-Feng Du<sup>*</sup></strong>; Predicting Golgi-resident protein types using pseudo amino acid compositions: Approaches with positional specific physicochemical properties, <em>Journal of Theoretical Biology</em>, 2016, 391: 35-42. <strong>(SCI)</strong>
</div><div class='outdent'>[14]&nbsp;&nbsp;<strong>Pufeng Du<sup>*</sup></strong>; Lusheng Wang<sup>*</sup>; Predicting human protein subcellular locations by the ensemble of multiple predictors via protein-protein interaction network with edge clustering coefficients, <em>PLoS ONE</em>, 2014, 9(1): e86879. <strong>(SCI)</strong>
</div><div class='outdent'>[13]&nbsp;&nbsp;<strong>Pufeng Du<sup>*</sup></strong>; Shuwang Gu; Yasen Jiao; PseAAC-General: fast building various modes of general form of Chou’s pseudo-amino acid composition for large-scale protein datasets, <em>International Journal of Molecular Sciences</em>, 2014, 15(3): 3495-3506. <strong>(SCI)</strong>
</div><div class='outdent'>[12]&nbsp;&nbsp;<strong>Pu-Feng Du</strong>; Chao Xu<sup>*</sup>; Predicting multisite protein subcellular locations: progress and challenges, <em>Expert Review of Proteomics</em>, 2013, 10(3): 227-237. <strong>(SCI)</strong>
</div><div class='outdent'>[11]&nbsp;&nbsp;<strong>Pufeng Du<sup>*</sup></strong>; Tingting Li; Xin Wang; Chao Xu; SubChlo-GO: predicting protein subchloroplast locations with weighted gene ontology scores, <em>Current Bioinformatics</em>, 2013, 8(2): 193-199. <strong>(SCI)</strong>
</div><div class='outdent'>[10]&nbsp;&nbsp;<strong>Pufeng Du<sup>*</sup></strong>; Yuan Yu; SubMito-PSPCP: predicting protein submitochondrial locations by hybridizing positional specific physicochemical properties with pseudoamino acid compositions, <em>BioMed Research International</em>, 2013, 2013: 263829. <strong>(SCI)</strong>
</div><div class='outdent'>[9]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du<sup>*</sup></strong>; Yang Tian; Yan Yan; Subcellular localization prediction for human internal and organelle membrane proteins with projected gene ontology scores, <em>Journal of Theoretical Biology</em>, 2012, 313: 61-67. <strong>(SCI)</strong>
</div><div class='outdent'>[8]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du<sup>*</sup></strong>; Xin Wang; Chao Xu; Yang Gao<sup>*</sup>; PseAAC-Builder: a cross-platform stand-alone program for generating various special Chou’s pseudo-amino acid compositions, <em>Analytical Biochemistry</em>, 2012, 425(2): 117-119. <strong>(SCI)</strong>
</div><div class='outdent'>[7]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du<sup>*</sup></strong>; Tingting Li; Xin Wang; Recent progress in predicting protein sub-subcellular locations, <em>Expert Review of Proteomics</em>, 2011, 8(3): 391-404. <strong>(SCI)</strong>
</div><div class='outdent'>[6]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du<sup>#</sup></strong>; Shengjiao Cao<sup>#</sup>; Yanda Li<sup>*</sup>; SubChlo: predicting protein subchloroplast locations with pseudo-amino acid composition and the evidence-theoretic K-nearest neighbor (ET-KNN) algorithm, <em>Journal of Theoretical Biology</em>, 2009, 261(2): 330-335. <strong>(SCI)</strong>
</div><div class='outdent'>[5]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du<sup>#</sup></strong>; Liyan Jia<sup>#</sup>; Yanda Li<sup>*</sup>; CURE-Chloroplast: a chloroplast C-to-U RNA editing predictor for seed plants, BMC Bioinformatics, 2009, 10(1): 135. <strong>(SCI)</strong>
</div><div class='outdent'>[4]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du</strong>; Yanda Li<sup>*</sup>; Prediction of C-to-U RNA editing sites in plant mitochondria using both biochemical and evolutionary information, <em>Journal of Theoretical Biology</em>, 2008, 253(3): 579-586. <strong>(SCI)</strong>
</div><div class='outdent'>[3]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du<sup>#</sup></strong>; Tao He<sup>#</sup>; Yanda Li<sup>*</sup>; Prediction of C-to-U RNA editing sites in higher plant mitochondria using only nucleotide sequence features, <em>Biochemical and Biophysical Research Communications</em>, 2007, 358(1): 336-341. <strong>(SCI)</strong>
</div><div class='outdent'>[2]&nbsp;&nbsp;&nbsp;&nbsp;Tao He<sup>#</sup>; <strong>Pufeng Du<sup>#</sup></strong>; Yanda Li<sup>*</sup>; dbRES: a web-oriented database for annotated RNA editing sites, Nucleic Acids Research, 2007, 35(Database Issue): D141-D144. <strong>(SCI)</strong>
</div><div class='outdent'>[1]&nbsp;&nbsp;&nbsp;&nbsp;<strong>Pufeng Du</strong>; Yanda Li<sup>*</sup>; Prediction of protein submitochondria locations by hybridizing pseudo-amino acid composition with various physicochemical features of segmented sequence, <em>BMC Bioinformatics</em>, 2006, 7(1): 518. <strong>(SCI)</strong>
</div><p class='vspace'><strong><sup>*</sup>：通讯作者；<sup>#</sup>：贡献相同。</strong>
</p>
<div class='vspace'></div>
</div>

                </div>
                <div class="clearer">&nbsp;</div>
            </div>
        </div>

        <div id="footer" style="border: 1px none #000;">
            <div id="copyright">
                <span style="float: left; padding-left: 20px;">
                    Contents Copyright © Pufeng Du. — All Rights Reserved
                </span>
                <span style="float: right; display: block; padding-right: 20px;">
                    <span id='sitepoweredby' title='Powered by PmWiki'>
    Powered by <a href='http://www.pmichaud.com/wiki/PmWiki/PmWiki' title='PmWiki Home'>PmWiki</a>
</span>
                </span>
            </div>
        </div>
    </body>
</html>
